.github/workflows/main.yml

```diff
@@ -266,19 +266,13 @@ jobs:
           python-version: '3.10'
           architecture: x64
 
-      - uses: actions/download-artifact@v3
-        with:
-          name: wheelhouse
-          path: wheelhouse
+      - uses: actions/download-artifact@v4
 
-      - uses: actions/download-artifact@v3
-        with:
-          name: version
-          path: .
```

Comment:
@mgovers this part should be recovered. We do not download vision file, so get a weired tag.